modal.js

```diff
@@ -0,0 +1,247 @@
+const body2 = document.querySelector('body');
+const aboutMe = document.querySelector('.about-me');
+const cardContainer = document.createElement('section');
+cardContainer.classList.add('card-works');
+const buttonsArr = [];
+const cardArr = [];
+const modalArr = [];
+const closeModalArr = [];
+
+const projects = [
+  {
+    name: 'Tonic',
+    featuredImage: "url('./images/Snapshoot-Portfolio-1.png')",
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    languagesList: ['html', 'css', 'javascript'],
+    liveLink: 'https://fikerte-t.github.io/Microverse-Student-Project-1/',
+    sourceLink: 'https://github.com/Fikerte-T/Microverse-Student-Project-1/',
+  },
+  {
+    name: 'Multi-Post Stories',
+    featuredImage: "url('./images/Snapshoot-Portfolio-2.png')",
```

Comment:
- [ ] For the project images, it is much better to add the featuredImage key as an object, for this reason, you would have an easier option of adding the alt attribute.
```
featureImage: {link: "url('./images/Snapshoot-Portfolio-1.png')", alt: "Tonic project image"}
```